infrastructure/analytics/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.7'
```

Comment:
Is it worth renaming this so we know that it is a docker-compose for plausible in case we add further options down the lines such as fathom

Reply:
renaming is probably a good idea - might be enough to create a plausible folder inside analytics - so if you want to stand up fathon then maybe a fathom folder could be added. Plausible adds a bunch of dependencies such as MailHog, Postgres etc.

Reply:
Have created an infrastructure/analytics/plausible folder structure.